pulseguardian/guardian.py

```diff
@@ -219,22 +230,66 @@ def back_to_normal_email(self, user, queue_data):
            queue_data['messages'], self.del_queue_size)
 
         if self.emails and user.email is not None:
-            sendemail(subject=subject, from_addr=config.email_from,
-                      to_addrs=[user.email], username=config.email_account,
-                      password=config.email_password, text_data=body,
-                      server=config.email_smtp_server,
-                      port=config.email_smtp_port,
-                      use_ssl=config.email_ssl)
+            self._sendemail(
+                subject=subject, to_addrs=[user.email], text_data=body)
+
+    def notify_connection_error(self):
+        if self.emails and not self._connection_error_notified:
+            ex_details = traceback.format_exc()
+            admin_emails = [user.email 
+                for user in User.query.filter_by(admin=True)]
+            subject = "Pulse error: Can't connect to Pulse"
+            body = '''Could not connect to Pulse.
+Details:
+Rabbit Url: {0}
+Rabbit User: {1}
+Error:
+{2}
+'''.format(config.rabbit_management_url, config.rabbit_user, ex_details)
+
+            self._sendemail(
+                subject=subject, to_addrs=admin_emails, text_data=body)
+            logging.error(subject + body)
```

Comment:
We should be logging at least a summary (maybe not the whole traceback) each time this fails, even if we have already sent out a notification, just so it's obvious from looking at the logs whether the problem is still ongoing.